tests_v2/test_focus_history_shortcut_cross_workspace.py

```
@@ -0,0 +1,102 @@
+
+
+def _must(cond: bool, msg: str) -> None:
+    if not cond:
+        raise cmuxError(msg)
+
+
+def _selected_workspace(c: cmux) -> str:
+    for _idx, wsid, _title, selected in c.list_workspaces():
+        if selected:
+            return wsid
+    raise cmuxError("no selected workspace")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)